README.md

```diff
@@ -27,7 +27,9 @@ https://codesoapbox.dev/keycloak-user-migration
 
 You must provide two REST endpoints (GET and POST) in your legacy authentication system under the URI `${restClientUri
 }/{$username}`, where `${restClientUri}` is a configurable base URL for the endpoints and `{$username}` is the
-username of the user that is attempting to sign in.
+username of the user that is attempting to sign in. It is possible to configure the plugin to use the legacy `userId`
```

Comment:
`Legacy 'userId'` is a bit confusing IMO, especially since it's written as a variable. I think it could use a slightly better explanation, perhaps something like:
`It is possible to configure the plugin to use a legacy user id (separate from the username) when making the credential verification request - use this option if your legacy system allows for changing usernames`.
It's a small change, but I think it will help understand the feature better.

`However, this option should only be enabled when the legacy user ids are migrated to Keycloak` - Perhaps it's just that it's late but I'm actually unsure what you mean by that. Aren't they never migrated and only optionally used for identification when migrating?

Reply:
Users can be migrated in such way that they keep their ids from the legacy system. The only thing that's required is to return the user `id` in the payload of the `GET` request. E.g.:
```
GET http://www.old-legacy-system.com/auth/todd@gmail.com

{
  "id": "12345",
  "username": "todd@gmail.com"
  ...
}
```

The user that will be created by keycloak will have a user id `12345`.

This is a feature that I contributed few months ago.

Reply:
Ok, I think I now understand what you meant by the last sentence :)
Still, I think the altered previous paragraph with `use this option if your legacy system allows for changing usernames` works better in this case.
Do you agree or am I missing some important detail?